### PR TITLE
#17 Adds the ability to use an alias on object queries.

### PR DESCRIPTION
**Pull request by paul-crashley**
* Updates Query class to allow an alias to be set
* Updates QueryBuilder class to allow for an alias to be set
* Updates tests to cover alias changes

### CODE REVIEW

File: tests/QueryBuilderTest.php [hunk 1]
```diff
@@ -38,11 +38,31 @@ public function setUp(): void
      */
     public function testConstruct()
     {
-        $builder = new QueryBuilder('Object');
+        $builder = new QueryBuilder('Object', 'ObjectAlias');
```
Comment (mghoneimy) [inline tests/QueryBuilderTest.php:41]: I would rather if we move this to a separate test to keep the `testConstruct` for the basic base case.

**Review by mghoneimy — APPROVED** [review]
I'm very impressed by the quality of this code review. Great documentation in the change log and read me files, great code, and great tests! I just added a minor comment to one of the tests before we can merge this.
Will merge this PR as soon as you address this minor change.

File: tests/QueryBuilderTest.php (hunk 1 above)
Comment (paul-crashley) [reply tests/QueryBuilderTest.php:59]: No problem. That is done for you now. Sorry about the delay.

**Comment by paul-crashley** [comment]
Updates made as requested

**Review by mghoneimy — APPROVED** [review]

**Comment by mghoneimy** [comment]
@paul-crashley Thank you and congrats on your contribution to this package :smile:

**Comment by mghoneimy** [comment]
Closes #17

**Pull request merged**